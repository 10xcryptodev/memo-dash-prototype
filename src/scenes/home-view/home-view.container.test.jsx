import React from 'react'
import configureStore from 'redux-mock-store'
import { shallow } from 'enzyme'
import HomeViewContainer from './home-view.container'

describe('<HomeViewContainer />', () => {
  let store

  beforeEach(() => {
    // Mock store
    const mockStore = configureStore()
    store = mockStore({
      user: { currentUser: { username: 'Username' } }
    })

    const div = document.createElement('div')
    document.body.appendChild(div)
  })

  describe('Shallow rendering', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<HomeViewContainer />, { context: { store } })
      expect(wrapper).toMatchSnapshot()
    })
  })
})
