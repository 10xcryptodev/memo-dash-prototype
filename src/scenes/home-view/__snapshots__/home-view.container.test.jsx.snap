// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeViewContainer /> Shallow rendering renders without crashing 1`] = `
<HomeViewComponent
  currentUser={
    Object {
      "username": "Username",
    }
  }
/>
`;
